docs/game_sdk/Discord_Voice.md

```diff
@@ -4,7 +4,9 @@
 > Need help with the SDK? Talk to us in the [Discord Developers Server](https://discord.gg/discord-developers)!
 
 > danger
-> Selling SKUs on Discord has now been discontinued as of March 1, 2022. [Read here for more info.](https://support-dev.discord.com/hc/en-us/articles/6309018858647-Self-serve-Game-Selling-Deprecation)
```

Comment:
Hmmm I think we may wanna keep the SKU stuff in all of this for now. When we get closer to the decommission date, we'll do a bigger sweep of all of these docs at which point it may make more sense.

Reply:
i removed it from the pages that are for any game and aren't related to the store